phalcon/mvc/model/query/builder.zep

```diff
@@ -379,7 +379,7 @@ class Builder implements BuilderInterface, InjectionAwareInterface
 	 *	$builder->join('Robots');
 	 *	$builder->join('Robots', 'r.id = RobotsParts.robots_id');
 	 *	$builder->join('Robots', 'r.id = RobotsParts.robots_id', 'r');
-	 *	$builder->join('Robots', 'r.id = RobotsParts.robots_id', 'r', 'LEFT');
+	 *	$builder->join('Robots', 'r.id = RobotsParts.robots_id', 'r', 'INNER');
```

Comment:
Line 376: (https://github.com/phalcon/cphalcon/blob/2.0.0/phalcon/mvc/model/query/builder.zep#L376)
"Adds a INNER join to the query"